web/templates/pages/tasks.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'base.html' %}
+
+{% block title %}All CodeCatz Tasks{% endblock %}
+
+{% block content%}
+
+from django import forms
+from .models import Task
+
+	<header class="row">
+		<div class="col-lg-6 col-sm-7">
+		      <h1>All CodeCatz Tasks</h1>
+		</div>
+		<div class="col-lg-6 col-sm-5">
+			<p>This page will show all tasks and who they are assigned to.</p>
+		</div>
+	</header>
+
+	<div class="row">
+	{% for task in task_list %}
```

Comment:
OK, then let's pretend there is a summer outside since we are into pretending...

But I can imagine a case where there will be a project, like Trackcat once was, where there are people talking about ... let's say the moon and the stars, for two weeks and being busy stargazing, no one have defined any tasks yet.

The least you can do, is to display a message that there is not task for this project